pyhf/__init__.py

```diff
@@ -1,4 +1,6 @@
 from . import tensor, optimize
+from .version import __version__
+__all__ = ['__version__']  # Used to satisfy pyflakes
```

Comment:
there's already an `__all__` in `__init__.py` -- you should add it in there. This will just get overridden.

Reply:
Yes. Sorry. Was rushing.